package com.my.dao;

import java.sql.Connection;
import java.sql.PreparedStatement;
import java.sql.SQLException;
import java.util.List;

import com.my.exception.AddException;
import com.my.exception.FindException;
import com.my.exception.ModifyException;
import com.my.exception.RemoveException;
import com.my.sql.MyConnection;
import com.my.vo.Customer;

public class CustomerDAOOracle implements CustomerDAO {

	@Override
	public void insert(Customer c) throws AddException {
		Connection con = null;
		try {
			con = MyConnection.getConnection();
		} catch (Exception e) {
			e.printStackTrace();
			throw new AddException("고객 추가 실패: 이유=" + e.getMessage());
		}
		PreparedStatement pstmt = null;
		String insertSQL = "INSERT INTO customer(id, pwd, name, zipcode, addr) VALUES (?,?,?,?,?)";
		try {
			pstmt = con.prepareStatement(insertSQL);
		} catch (SQLException e) {
			e.printStackTrace();
		}
		
	}

	@Override
	public List<Customer> selectAll() throws FindException {
		// TODO Auto-generated method stub
		return null;
	}

	@Override
	public Customer selectById(String id) throws FindException {
		// TODO Auto-generated method stub
		return null;
	}

	@Override
	public Customer update(Customer c) throws ModifyException {
		// TODO Auto-generated method stub
		return null;
	}

	@Override
	public Customer delete(String id) throws RemoveException {
		// TODO Auto-generated method stub
		return null;
	}

}
